cli/cli.go

```diff
@@ -94,6 +94,71 @@ func main() {
 				},
 			},
 		},
+		{
+			Name: "delete",
+			Subcommands: []cli.Command{
+				{
+					Name:      "pipeline",
+					Usage:     "delete pipeline",
+					ArgsUsage: "<pipeline-name>",
+					Before:    requireNameArg,
+					Action:    deletePipeline,
+				},
+				{
+					Name:      "build",
+					Usage:     "delete build",
+					ArgsUsage: "<pipeline-name>",
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:  "build, b",
+							Usage: "build number, if not provided will not proceed deletion",
+						},
+					},
+					Before: requireNameArg,
+					Action: deleteBuild,
+				},
+			},
+		},
+		{
+			Name:  "deploy",
+			Usage: "deploy kontinuous app in the cluster",
+			Subcommands: []cli.Command{
+				{
+					Name:   "remove",
+					Usage:  "remove kontinuous app in the cluster",
+					Action: removeDeployedApp,
+				},
+			},
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "namespace",
+					Usage: "Required, Kubernetes namespace to deploy kontinuous",
+					Value: "kontinuous",
+				},
+				cli.StringFlag{
+					Name:  "s3-access-key",
+					Usage: "Required, S3 access key",
+				},
+				cli.StringFlag{
+					Name:  "s3-secret-key",
+					Usage: "Required, S3 secret key",
+				},
```

Comment:
Earlier I was going to suggest that we also add a s3 URL option alongside the keys for the feature request of using external object stores (S3/GCS/etc). May be better to disable the keys for now so we can offer in future.